2024-07-20 16:29:39,154 INFO    MainThread:3748401 [wandb_setup.py:_flush():76] Configure stats pid to 3748401
2024-07-20 16:29:39,154 INFO    MainThread:3748401 [wandb_setup.py:_flush():76] Loading settings from /home/pi/.config/wandb/settings
2024-07-20 16:29:39,155 INFO    MainThread:3748401 [wandb_setup.py:_flush():76] Loading settings from /home/pi/Downloads/LocomotionWithNP3O-master/wandb/settings
2024-07-20 16:29:39,155 INFO    MainThread:3748401 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-07-20 16:29:39,155 INFO    MainThread:3748401 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'train.py', 'program': './train.py'}
2024-07-20 16:29:39,155 INFO    MainThread:3748401 [wandb_init.py:_log_setup():461] Logging user logs to /home/pi/Downloads/LocomotionWithNP3O-master/wandb/run-20240720_162939-v84oys7t/logs/debug.log
2024-07-20 16:29:39,155 INFO    MainThread:3748401 [wandb_init.py:_log_setup():462] Logging internal logs to /home/pi/Downloads/LocomotionWithNP3O-master/wandb/run-20240720_162939-v84oys7t/logs/debug-internal.log
2024-07-20 16:29:39,155 INFO    MainThread:3748401 [wandb_init.py:init():495] calling init triggers
2024-07-20 16:29:39,155 INFO    MainThread:3748401 [wandb_init.py:init():498] wandb.init called with sweep_config: {}
config: {}
2024-07-20 16:29:39,155 INFO    MainThread:3748401 [wandb_init.py:init():548] starting backend
2024-07-20 16:29:39,155 INFO    MainThread:3748401 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-07-20 16:29:39,173 INFO    MainThread:3748401 [backend.py:ensure_launched():217] starting backend process...
2024-07-20 16:29:39,184 INFO    MainThread:3748401 [backend.py:ensure_launched():222] started backend process with pid: 3749096
2024-07-20 16:29:39,185 INFO    MainThread:3748401 [wandb_init.py:init():558] backend started and connected
2024-07-20 16:29:39,189 INFO    MainThread:3748401 [wandb_init.py:init():634] updated telemetry
2024-07-20 16:29:39,189 INFO    MainThread:3748401 [wandb_init.py:init():665] communicating run to backend with 30 second timeout
2024-07-20 16:29:40,384 INFO    MainThread:3748401 [wandb_run.py:_on_init():1919] communicating current version
2024-07-20 16:29:45,384 INFO    MainThread:3748401 [wandb_run.py:_on_init():1923] got version response None
2024-07-20 16:29:45,384 INFO    MainThread:3748401 [wandb_init.py:init():700] starting run threads in backend
2024-07-20 16:29:46,672 INFO    MainThread:3748401 [wandb_run.py:_console_start():1893] atexit reg
2024-07-20 16:29:46,673 INFO    MainThread:3748401 [wandb_run.py:_redirect():1766] redirect: SettingsConsole.REDIRECT
2024-07-20 16:29:46,673 INFO    MainThread:3748401 [wandb_run.py:_redirect():1771] Redirecting console.
2024-07-20 16:29:46,674 INFO    MainThread:3748401 [wandb_run.py:_redirect():1827] Redirects installed.
2024-07-20 16:29:46,674 INFO    MainThread:3748401 [wandb_init.py:init():732] run started, returning control to user process
2024-07-20 16:35:46,353 INFO    MainThread:3748401 [wandb_run.py:_atexit_cleanup():1862] got exitcode: 255
2024-07-20 16:35:46,354 INFO    MainThread:3748401 [wandb_run.py:_restore():1834] restore
2024-07-20 16:35:48,559 INFO    MainThread:3748401 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 519
  total_bytes: 519
}

2024-07-20 16:35:49,239 INFO    MainThread:3748401 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 519
  total_bytes: 519
}

2024-07-20 16:35:49,491 INFO    MainThread:3748401 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 519
  total_bytes: 567141
}

2024-07-20 16:35:49,593 INFO    MainThread:3748401 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 519
  total_bytes: 569348
}

2024-07-20 16:35:49,696 INFO    MainThread:3748401 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 519
  total_bytes: 569348
}

